CMakeLists.txt

```diff
@@ -79,8 +80,13 @@ if (DYNAFLOW_LAUNCHER_SHARED_LIB)
 endif()
 message(STATUS "Compile internal libraries in " ${LINK_TYPE})
 
+
+
 set(CMAKE_CXX_FLAGS_DEBUG "-D_DEBUG_=1")
 set(CMAKE_CXX_FLAGS "-DLANG_CXX11")
+if(FORCE_CXX11_ABI)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=1")
```

Comment:
What the point of adding the ABI flag if all projects are compiled in the same container ?

Reply:
It was proposed by @gautierbureau when we decided to prepare the builds for CI of dependant projects.

Reply:
Maybe the container we are using requires it ?

Reply:
This flag is used for compatibility between old compilers and new compilers but if all are using the same version (i.e. the same container) I don't think it is required. TBC